index.html

```diff
@@ -1,79 +1,86 @@
-        <div class="container-fluid">
-            <a class="navbar-brand" href="index.html">Ma page</a>
-            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
-                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
-                aria-label="Toggle navigation">
-                <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarSupportedContent">
-                <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-                    <li class="nav-item">
-                        <a class="nav-link active" aria-current="page" href="index.html">Home</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="about.html">À propos</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="contact.html">Contact</a>
-                    </li>
-                </ul>
-            </div>
+      <div class="container-fluid">
+        <a class="navbar-brand" href="index.html">LOGO</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup"
+          aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+            <a class="nav-link active" aria-current="page" href="index.html">Accueil</a>
+            <a class="nav-link" href="apropos.html">À propos</a>
+            <a class="nav-link" href="mesprojets.html">Mes Projets</a>
+            <a class="nav-link" href="contact.html">Contact</a>
+          </div>
         </div>
+      </div>
     </nav>
-
-    <div class="container mt-3">
-        <h1>Bienvenue sur ma page Bootstrap</h1>
-        <p>Cette page est créée avec Bootstrap.</p>
+  </header>
+  <main>
+    <nav class="mt-2" style="--bs-breadcrumb-divider: '>';" aria-label="breadcrumb">
+      <ol class="breadcrumb mx-2">
+        <li class="breadcrumb-item"><a class="text-success" href="index.html">Accueil</a></li>
+        <li class="breadcrumb-item active" aria-current="page">À propos</li>
+      </ol>
+    </nav>
+    <h1 class="text-success text-center mt-5 mb-5 fs-3">Accueil</h1>
+    <div class="card m-5" style="max-width: 100%;">
+      <div class="row g-0">
+        <div class="col-md-4">
+          <img src="https://picsum.photos/400/200" class="img-fluid rounded-start" alt="Profile picture">
+        </div>
+        <div class="col-md-8">
+          <div class="card-body">
+            <h5 class="card-title mt-4">Résumé</h5>
+            <p class="card-text mt-3">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Possimus dolorem
```

Comment:
Notice that your image is not aligned with the card title. The same is happening in other places as well.

![image](https://user-images.githubusercontent.com/6362750/235264110-78f9d625-7b6d-48ed-adbd-9b0a812e9f52.png)

Reply:
Img was not responsive, test was done with iphone screen size and mac screen size only. 
Done.